cloudevents/sdk/event/base.py

```diff
@@ -159,5 +158,4 @@ def MarshalBinary(
             headers["ce-{0}".format(key)] = value
 
         data, _ = self.Get("data")
-        return headers, io.BytesIO(
-            str(data_marshaller(data)).encode("utf-8"))
+        return headers, data_marshaller(data)
```

Comment:
This changes the output to be the result of the data_marshaller, rather than enforcing that the data is utf-8 encoded.

Not sure if this is intentional or not.

Reply:
my original idea doesn't really work well, so, I think it would be simpler to let people decide what they want to do with the event data.
for instance, if you'd like to get an `io.Bytes` from marshaller you can do the following:

```python
m.ToRequest(event, lambda x: io.BytesIO(str(x).encode("utf-8")))
```
but forcing them to do things they don't want is a bad idea, though.